### DISCUSSION TITLE
老铁，我建议你去油管搞个频道，还有推特搞个账号

### DISCUSSION THREAD

**Discussion by KenNaNa** [apache/hertzbeat#1110] [Ideas]
老铁，我建议你去油管搞个频道，还有推特搞个账号

这是我的各人建议，我也被裁员了

这是我目前做的一个工具包，可以的话，可以把我放在你的展示的地方？

kennana.github.io/toolkit-use/

**Comment by tomsun28** [comment]
谢谢老哥建议 😉😉😉😉😉
